src/stories/Blocks/page/Page.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import Page from "./Page";
+
+export default {
+  title: "Blocks / Pages",
+  component: Page,
+  decorators: [withDesign],
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?type=design&node-id=7477-39243&mode=dev",
+    },
+  },
+  argTypes: {
+    hero: {
+      control: "object",
+      description: "Object containing hero details",
+    },
+  },
+} as ComponentMeta<typeof Page>;
+
+const Default: ComponentStory<typeof Page> = (args) => <Page {...args} />;
+
+export const frontPage = Default.bind({});
```

Comment:
Just marking this here. 

I think it's important we show the "issues" with alot of text on this component, some breakpoints dont support it very well. 

![image](https://github.com/danskernesdigitalebibliotek/dpl-design-system/assets/13272656/aaac61b6-ee1c-4f72-af6e-348b9d8c42f5)

Reply:
good point 👍